servicetalk-buffer-api/build.gradle

```diff
@@ -16,15 +16,20 @@
 
 apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
 
+test {
```

Comment:
This can be removed, its provided from the `servicetalk-gradle-plugin-internal-library`